frontend/helpers/env-vars.js

```diff
@@ -11,10 +11,10 @@ function isBrowser() {
  * variable from the server (process.env).
  */
 export function env(key) {
-  if (isBrowser()) {
-    // eslint-disable-next-line no-underscore-dangle
-    return window.__ENV[key];
-  }
+  // if (isBrowser()) {
+  //   // eslint-disable-next-line no-underscore-dangle
+  //   return window.__ENV[key];
+  // }
 
-  return process.env[key];
+  return 'https://tdt.akvotest.org/cms';
```

Comment:
Why does this have to be hard-coded? This will not work in a different environment.